### ISSUE TITLE
Date representations are also US-centric

### ISSUE THREAD

**Issue by farski** [PRX/publish.prx.org#458]
eg 09/04/2017 would be confusing for European users

**Comment by pkarman** [comment]
@farski once you've given this an `Estimate` feel free to move it to Backlog.

**Comment by farski** [comment]
@pkarman I don't think I could give a reasonable estimate for this one

**Comment by pkarman** [comment]
@kookster you moved this to Backlog - do you have an estimate?

**Comment by sandikbarr** [comment]
It's difficult to give this an estimate without an acceptance criteria. Should we start an epic for internationalization and think about what the requirements are as a whole?

**Comment by pkarman** [comment]
While i18n discussion might be worthwhile vis-a-vis business needs, this particular issue seems of a much smaller scope: the display of dates. If it's a DateTime, there are also time and timezone implications (e.g. always show UTC? or the user's browser TZ?).

I'm guessing the reluctance to give an estimate is that it's unclear what the scope of the problem is. Maybe we need a discovery ticket, to determine how big the problem is?

**Comment by pkarman** [comment]
I've added some proposed acceptance criteria to the issue description.

**Comment by farski** [comment]
FWIW I don't think 8601 ever uses slashes. I also didn't mean for this to be as large as i18n support; just anything that isn't so ambiguous. YMD is a good option. `Oct 5 2017` also feels acceptable, even if EU people would be more accustomed to `5 Oct 2017` since it's not ambiguous.

**Comment by pkarman** [comment]
@farski you are correct - 8601 does not use slashes. I was trying for parity with existing. I have changed the requirements to `-` instead.
